com.sap.cloud.lm.sl.cf.process/src/test/java/com/sap/cloud/lm/sl/cf/process/steps/DetermineServiceCreateUpdateServiceActionsStepTest.java

```diff
@@ -163,6 +165,8 @@ private void prepareClient() {
                 .thenReturn(existingService);
             Mockito.when(client.getServiceInstance(stepInput.existingService.getName(), false))
                 .thenReturn(existingServiceInstance);
+            Mockito.when(client.getServiceParameters(existingService.getMetadata().getGuid()))
```

Comment:
Why there is not positive test where service parameters are present?

Reply:
Done